### PR TITLE
Fixed some jsonrpc methods

### PR DESCRIPTION
**Pull request by ZeljkoBenovic**
# Description

This PR fixes / implements 3 broken jsonrpc methods:
* eth_chainID
    * changed method from eth_chainId to eth_chainID in polygon-sdk public docs
* net_peerCount
    * impemented a proper method that returns the number of peers
* web3_clientVersion
   * fixed output to accomodate json responce
# Changes include

- [x] Bugfix (non-breaking change that solves an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

# Breaking changes

Please complete this section if any breaking changes have been made, otherwise delete it

# Checklist

- [x] I have assigned this PR to myself
- [x] I have added at least 1 reviewer
- [x] I have tested this code
- [x] I have updated the README and other relevant documents (guides...)
- [x] I have added sufficient documentation both in code, as well as in the READMEs

# Additional comments

Please post additional comments in this section if you have them, otherwise delete it

### CODE REVIEW

File: server/server.go [hunk 1]
```diff
@@ -351,11 +351,18 @@ type jsonRPCHub struct {
...
+	*network.Server
 	consensus.Consensus
 }
 
 // HELPER + WRAPPER METHODS //
 
+func (j *jsonRPCHub) GetPeers() int {
+	peers := j.Server.Peers()
+
+	return len(peers)
```
Comment (zivkovicmilos) [inline server/server.go]: You can inline this call:
```go
return len(j.Server.Peers())
```

File: version/version.go [hunk 2]
```diff
@@ -17,3 +17,8 @@ func GetVersion() string {
 
 	return version
 }
+
+// returns jsonrpc representation of the version
+func GetVersionJsonrpc() string {
+	return "v" + Version
```
Comment (zivkovicmilos) [inline version/version.go:23]: Do we know what kind of response clients like Ganache give for this RPC call?

**Review by zivkovicmilos — APPROVED** [review]
Thank you for fixing up these methods, it's easy to forget about these things in the midst of all the features 🙏 

I've left a couple of questions for discussion, but everything seems solid 💯

**Review by Kourin1996 — APPROVED** [review]

File: version/version.go (hunk 2 above)
Comment (lazartravica) [reply version/version.go:23]: This is up to the client. Polygon network returns this: `{"jsonrpc":"2.0","id":1,"result":"bor/v0.2.13-stable-15965124/linux-amd64/go1.17.6"}`

**Review by lazartravica — APPROVED** [review]

**Pull request merged**